src/k8s-engine/wsl.ts

```diff
@@ -1704,6 +1591,16 @@ CREDFWD_URL='http://${ hostIPAddr }:${ stateInfo.port }'
     return this.wslify(path.join(paths.resources, 'linux', 'wsl-helper'), distro);
   }
 
+  /**
+   * Return the Linux path to the vtunnel peer executable.
+   */
+  protected getVtunnelPeerPath(distro?: string): Promise<string> {
+    // We need to get the Linux path to our helper executable; it is easier to
+    // just get WSL to do the transformation for us.
+
+    return this.wslify(path.join(paths.resources, 'linux', 'vtunnel'), distro);
```

Comment:
This shuold be `resources/linux/internal`, but either way this doesn't seem to exist?